Makefile

```diff
@@ -154,18 +154,17 @@ cf-deploy: ## Deploys the app to Cloud Foundry
 	@cf app --guid ${CF_APP} || exit 1
 
 	# cancel any existing deploys to ensure we can apply manifest (if a deploy is in progress you'll see ScaleDisabledDuringDeployment)
-	cf v3-cancel-zdt-push ${CF_APP} || true
-
-	cf v3-apply-manifest ${CF_APP} -f <(make -s generate-manifest)
```

Comment:
Is there a reason you didn't use `cf apply-manifest` here instead of moving the manifest into the `cf push` statement? It looks like what you are proposing does roughly the same thing but I just wanted to double check there were no differences? It seems like a sensible change, just wondering if there is a reason we didn't do it like this already before

Reply:
yeah - i also read up on the docs. looks like it doesn't do anything separate and keeps it cleaner and simpler to reason about.

i know andy had a big masterplan around breaking up the deploy process into its separate steps, and then reusing the same droplet with different manifests per environment, so i wonder if we originally did it to try that.

i'd be keen to revisit that some time, especially as cf v7 makes those commands mainline rather than bodging it with cf curl commands (see https://docs.cloudfoundry.org/devguide/push-sub-commands.html). But that would be a significant task and would need us to re-architect the pipelines to keep track of droplet IDs etc. (benefit: guaranteed identical code on diff environments, quicker deploys, quicker rollbacks)